.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
I think `yarn.lock` is more thorough.  If we do a `yarn upgrade <pkg>`, it may not need to touch `package.json`, but `yarn lock` may have changed.